kotlinx-coroutines-core/jvm/src/debug/internal/DebugProbesImpl.kt

```diff
@@ -234,17 +236,17 @@ internal object DebugProbesImpl {
     /*
      * Internal (JVM-public) method used by IDEA debugger as of 1.4-M3.
      */
-    public fun dumpCoroutinesInfo(): List<DebugCoroutineInfo> =
+    fun dumpCoroutinesInfo(): List<DebugCoroutineInfo> =
         dumpCoroutinesInfoImpl { owner, context -> DebugCoroutineInfo(owner.info, context) }
 
     /*
      * Internal (JVM-public) method to be used by IDEA debugger in the future (not used as of 1.4-M3).
      * It is equivalent to [dumpCoroutinesInfo], but returns serializable (and thus less typed) objects.
      */
-    public fun dumpDebuggerInfo(): List<DebuggerInfo> =
+    fun dumpDebuggerInfo(): List<DebuggerInfo> =
         dumpCoroutinesInfoImpl { owner, context -> DebuggerInfo(owner.info, context) }
 
-    public fun dumpCoroutines(out: PrintStream): Unit = synchronized(out) {
+    internal fun dumpCoroutines(out: PrintStream): Unit = synchronized(out) {
```

Comment:
The probability of this frame appearing in user's dumps is quite high, would be nice to avoid mangling here with `@JvmName("dumpCoroutines")`